.circleci/config.yml

```diff
@@ -146,7 +146,10 @@ jobs:
     resource_class: large
     steps:
       - checkout
-      - run: make tools
+      - run: |
+          make tools
+          # restore any go.{mod,sum} files that were updated indirectly
+          git reset --hard HEAD
```

Comment:
This works perfectly fine since make tools just does a bunch of go getting. If we ever modify it in the future, maybe setting it to checkout `go.{mod,sum}` specifically is better.